src/course_bot/essay.clj

```diff
@@ -152,12 +154,12 @@
       :approve
       (fn [tx {{id :id} :from text :text} {essay-text :essay-text}]
         (talk/when-parse-yes-or-no
```

Comment:
I think here (and in all code) we need to throw away `when-parse-yes-or-no` as to complicated. Current solution with `case + normalize-yes-no-text text` -- much better.

Maybe we need to extract `no` and `default` reaction into functions.